### PR TITLE
Fix icon imports in authentication

### PR DESCRIPTION
**Pull request by MonsieurRadiant**
Closes #  Github issue #324 

## Description

Fix icon imports in authetication

### CODE REVIEW

File: contents/page-sections/authentication/forgot-password/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ArrowLeft } from "@yamada-ui/lucide"
+import { ArrowLeft as ArrowLeftIcon } from "@yamada-ui/lucide"
```
Comment (Tomoya-Matsubara) [inline contents/page-sections/authentication/forgot-password/index.tsx:1]: `ArrowLeftIcon` can be imported without aliasing.

```suggestion
import { ArrowLeftIcon } from "@yamada-ui/lucide"
```

File: contents/page-sections/authentication/with-image/index.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Eye, EyeOff } from "@yamada-ui/lucide"
+import { Eye as EyeIcon, EyeOff as EyeOffIcon } from "@yamada-ui/lucide";
```
Comment (Tomoya-Matsubara) [inline contents/page-sections/authentication/with-image/index.tsx:1]: `EyeIcon` and `EyeOffIcon` can also be imported directly.

```suggestion
import { EyeIcon, EyeOffIcon } from "@yamada-ui/lucide"
```

File: contents/page-sections/authentication/with-title/index.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Eye, EyeOff } from "@yamada-ui/lucide"
+import { Eye as EyeIcon, EyeOff as EyeOffIcon } from "@yamada-ui/lucide";
```
Comment (Tomoya-Matsubara) [inline contents/page-sections/authentication/with-title/index.tsx:1]: Same here. `EyeIcon` and `EyeOffIcon` can be imported directly.

```suggestion
import { EyeIcon, EyeOffIcon } from "@yamada-ui/lucide"
```

**Comment by Tomoya-Matsubara** [comment]
LGTM! Thank you :)

**Review by Tomoya-Matsubara — APPROVED** [review]

**Review by zonoryo17 — APPROVED** [review]
LGTM👾

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**